nova_vm/src/ecmascript/builtins/text_processing/string_objects/string_prototype.rs

```diff
@@ -1478,3 +1513,44 @@ enum TrimWhere {
     End,
     StartAndEnd,
 }
+
+enum NormalizeForm {
+    NFC,
+    NFD,
+    NFKC,
+    NFKD,
+}
+
+impl NormalizeForm {
+    fn as_str(&self) -> &'static str {
```

Comment:
**nitpick:** Is this function actually ever used?

Reply:
Goot catch, I was using it in a previous implementation and forgot to remove it after refactoring it :)